docs/src/inputs/durationPicker/durationPicker.jsx

```diff
@@ -1,351 +1,199 @@
+                >
+                    <DurationPickerLabelExample />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="example"
+                        variant="h2"
+                    >
+                        Required Duration Picker
+                    </Heading>
 
-    _onDurationPicker3Change(newValue) {
-        this.setState({ value3: newValue });
-    }
+                    <Typography
+                        variant="body1"
+                    >
+                        You can use the
+                        <code>required</code>
+                        prop to mark the DurationPicker as a required input.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./durationPickerRequiredExample').default}
+                >
+                    <DurationPickerRequiredExample />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="example"
+                        variant="h2"
+                    >
+                        Required Duration Picker with Error Handling
+                    </Heading>
 
-    _onDurationPicker4Change(newValue) {
-        this.setState({ value4: newValue });
-    }
+                    <Typography
+                        variant="body1"
+                    >
+                        You can use the
+                        <code>error</code>
+                        prop to pass an error message.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./durationPickerErrorHandlingExample').default}
+                >
+                    <DurationPickerErrorHandlingExample />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="example"
+                        variant="h2"
+                    >
+                        Disabled Duration Picker
+                    </Heading>
 
-    _onDurationPicker5Change(newValue) {
-        this.setState({ value5: newValue });
-    }
+                    <Typography
+                        variant="body1"
+                    >
+                        You can use the
+                        <code>disable</code>
+                        prop to disable the DurationPicker input control.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./durationPickerDisabledExample').default}
+                >
+                    <DurationPickerDisabledExample />
+                </Example>
+
+                <ComponentApi
+                    docs={[
+                        durationPickerDoc,
+                    ]}
+                />
```

Comment:
Please add the ComponentVersionIdentifier here.

Reply:
Will do

Reply:
Hmm ... I don't think that is `dev` branch yet  ... It looks to be brand spanking new in `feature/10.0.0--grid-2.0.0` / #191 ... right?

Reply:
Well, it is also in `feature/doc_page_components` / #192 ... we definitely need to merge that guy first it looks like.

Reply:
Fixed in 629045a.

![image](https://user-images.githubusercontent.com/4349658/91437683-1b6d4e80-e81f-11ea-9c3d-80dfc4e70031.png)